packages/api/src/controllers/nominationController.js

```diff
@@ -75,6 +75,31 @@ const nominationController = {
     };
   },
 
+  /**
+   * This will add a new nomination to the table based from another controller using its transaction.
+   * @param {string} nominationType This is the workflow group name.
+   * @param {string} initialStatus This is the initial workflow status for the status log.
+   * @returns The created nomination and status row.
+   */
+  async addNominationFromController(nominationType, initialStatus, req, trx) {
+    const data = req.body;
+    data.nomination_type = nominationType;
+    //TODO: Hopefully this gets changed/removed with workflow ranking
+    const { workflow_id } = await NominationWorkflowModel.query(trx)
+      .select('workflow_id')
+      .where('status', initialStatus)
+      .andWhere('type_group', nominationType)
+      .first();
+    data.workflow_id = workflow_id;
+    const nomination = await baseController.postWithResponse(NominationModel, data, req, {
+      trx,
+    });
```

Comment:
You might go in a different direction, but `data`(line 97) needs to have `nomination_id`.
UI looks good!

Reply:
Ah thanks!, Looks like I accidentally dropped it in commit 4293d61

Reply:
See line 104 - linting likely moved it down a bit